### PR TITLE
Use a final object to lock on

### PR DESCRIPTION
**Pull request by matthias-ronge**
Java locks on _objects_ that are held in variables. It does _not_ lock on the variable. If the variable is changed in the `synchronized` block, a concurrent thread will lock on the other object, making the locking not working as expected.

Fix for Coverity IDs 44438, 44440

### CODE REVIEW

File: Goobi/src/de/sub/goobi/persistence/HibernateUtilOld.java [hunk 1]
```diff
@@ -37,6 +37,8 @@
 import org.hibernate.auction.exceptions.InfrastructureException;
 import org.hibernate.cfg.Configuration;
 
+import com.sun.org.apache.bcel.internal.generic.NEW;
+
```
Comment (henning-gerhardt) [inline Goobi/src/de/sub/goobi/persistence/HibernateUtilOld.java]: Is this import statement necessary?

**Comment by henning-gerhardt** [comment]
Did you search for similar (wrong) usage in project?

**Comment by matthias-ronge** [comment]
No. I fixed it because coverity reported it. If I saw it right, it reported only these two locations of the problem.

File: Goobi/src/de/sub/goobi/persistence/HibernateUtilOld.java (hunk 1 above)
Comment (matthias-ronge) [inline Goobi/src/de/sub/goobi/persistence/HibernateUtilOld.java]: No. I guess it came in by the Eclipse auto-complete feature when typing **new** Object()….
It’s ugly, sometimes.

**Pull request merged**